### PR TITLE
Add Song.requestedBy usage and remove example

### PR DESCRIPTION
**Pull request by thomasthebro1**
Add new properties and remove example to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -424,6 +426,32 @@ client.on('message', async (message) => {
...
+```js
+client.on('message', async (message) => {
+
+    const args = message.content.slice(settings.prefix.length).trim().split(/ +/g);
+    const command = args.shift().toLowerCase();
+
+    if(command === 'remove'){
+        // Removes a song from the queue
+        client.player.remove(message.guild.id, args[0]).then(() => {
+            message.channel.send('Removed song!')
```
Comment (Androz2091) [inline README.md:449]: Add missing semicolon
```suggestion
            message.channel.send('Removed song!');
```

File: README.md [hunk 2]
```diff
@@ -424,6 +426,32 @@ client.on('message', async (message) => {
 });
 ```
 
+### Remove
+To remove a song from the queue, use the `client.player.remove()` function.
+
+**Usage:**
+
+```js
+client.player.remove(guildID, song)
```
Comment (Androz2091) [inline README.md:435]: Add missing semicolon
```suggestion
client.player.remove(guildID, song);
```

File: README.md [hunk 3]
```diff
@@ -65,6 +65,8 @@ client.player.clearQueue(guildID);
 client.player.getQueue(guildID);
 // Skip the current song
 client.player.skip(guildID);
+// Remove a song from the queue using the index number
+client.player.remove(guildID, song)
```
Comment (Androz2091) [inline README.md:69]: Add missing semicolon
```suggestion
client.player.remove(guildID, song);
```

File: README.md [hunk 4]
```diff
@@ -124,8 +126,8 @@ client.on('message', async (message) => {
     // will play "Despacito" in the member voice channel
 
     if(command === 'play'){
-        let song = await client.player.play(message.member.voice.channel, args[0])
-        message.channel.send(`Currently playing ${song.name}!`);
+        let song = await client.player.play(message.member.voice.channel, args[0], message.member.user.tag);
+        message.channel.send(`Currently playing ${song.name}! - Requested by ${song.requestedBy}`)
```
Comment (Androz2091) [inline README.md:130]: Add missing semicolon
```suggestion
        message.channel.send(`Currently playing ${song.name}! - Requested by ${song.requestedBy}`);
```

**Comment by thomasthebro1** [comment]
Done.

**Pull request merged**